### DISCUSSION TITLE
Cannot get USB Serial working on STM32F407 Discovery board

### DISCUSSION THREAD

**Discussion by LAPSyLAB** [stm32duino/Arduino_Core_STM32#1308] [Q&A]

**Comment by fpistm** [comment]
Hi @RobRoz 
Some more information would be useful. Core version, Arduino IDE version, which options are used,...

I've tested the disco F407 and USB CDC works fine.
![image](https://user-images.githubusercontent.com/20641798/109124813-a5a15b80-774b-11eb-9190-eecffb522344.png)

Did you enabled the USB support in the"Tools" menu ?
If yes check your cable is well connected to the CN5 USB port.
